.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
[OPTIONAL]

- [ ] Although you've added node_modules to .gitignore, you still need to manually remove it from github. This is optional only because it was not part of the required changes in the last review. I strongly recommend that you do this